### PR TITLE
Smoke test

### LINKED ISSUE #5: Integration tests

### PR DESCRIPTION
**Pull request by nightkr**
Fixes #5

### CODE REVIEW

File: .dir-locals.el [hunk 1]
```diff
@@ -0,0 +1,4 @@
+;;; Directory Local Variables
```
Comment (soenkeliebau) [inline .dir-locals.el:1]: Is this supposed to be in here?

File: .github/workflows/maven.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    - uses: actions/checkout@v2
+    - name: Set up JDK 11
+      uses: actions/setup-java@v2
+      with:
+        java-version: '11'
+        distribution: 'adopt'
+        cache: maven
+    - name: Set up OpenPolicyAgent
+      run: |
+        curl -L -o opa https://openpolicyagent.org/downloads/v0.33.1/opa_linux_amd64_static
```
Comment (soenkeliebau) [inline .github/workflows/maven.yml:27]: I have no better suggestion, but hardcoding the version here feels somehow a bit ... dirty :)

File: src/test/java/tech/stackable/trino/opa/OpaAuthorizerTest.java [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+                                .PUT(HttpRequest.BodyPublishers
+                                        .ofString(stringOfLines(policyLines)))
+                                .header("Content-Type", "text/plain").build(),
+                        HttpResponse.BodyHandlers.ofString());
+        assertEquals(policyRes.statusCode(), 200, "Failed to submit policy: " + policyRes.body());
+    }
+
+    @Test
+    public void testShouldAllowQueryIfDirected() throws IOException, InterruptedException {
+        submitPolicy("package trino", "default can_execute_query = true",
```
Comment (soenkeliebau) [inline src/test/java/tech/stackable/trino/opa/OpaAuthorizerTest.java:135]: I have no strong objection to doing things this way, but it seems a bit cumbersome to create an indented multiline string.
Would maybe using test resources and reading from a file work as well?
That would have the drawback of not having the rule readable in the test though, which is significant ...

File: .dir-locals.el (hunk 1 above)
Comment (teozkr) [reply .dir-locals.el:1]: It documents what formatting profile is used, and is automatically picked up for Emacs users. I'd love to use a cross-editor file instead, but I'm not aware of any (EditorConfig is very barebones, comparatively).

File: src/test/java/tech/stackable/trino/opa/OpaAuthorizerTest.java (hunk 3 above)
Comment (teozkr) [reply src/test/java/tech/stackable/trino/opa/OpaAuthorizerTest.java:135]: Yeah.. it's definitely not pretty. But pulling in something like Scala or requiring a newer JDK just for multiline strings also feels overkill.. :/

Comment (soenkeliebau) [reply src/test/java/tech/stackable/trino/opa/OpaAuthorizerTest.java:135]: fine for me

**Pull request merged**